templates/contribute/pr-review.md

```diff
@@ -0,0 +1,715 @@
+You need spaces around `*`, `:` at the end of the line and the `rw` to 
+be on the same line.
+```suggestion
+theorem mul_assoc_assoc {α : Type _} [Semigroup α] (a b c d : α) :
+    a * b * c * d = a * (b * (c * d)) := by
+  rw [mul_assoc, mul_assoc]
+```
+````
+
+#### Naming conventions
+
+```lean
+theorem inv_is_unit_times_self_eq_1 {M : Type _} [Monoid M] {a : M} (h : IsUnit a) :
+    ↑(IsUnit.unit h)⁻¹ * a = 1 := sorry
+```
+
+The above lemma is taken directly from the library, can you guess its actual name?
+It is `IsUnit.inv_val_mul`. An appropriate review to suggest a new name might look
+something like:
+
+````markdown
+In order to accord with the 
+[naming conventions](https://leanprover-community.github.io/contribute/naming.html)
+for mathlib, I suggest renaming this to: `IsUnit.inv_val_mul`. Note that:
+
+- we use `mul` instead of `times`, and `one` instead of `1`
+- the lemma is sufficiently clear without the reference to `1`
+- If we are referencing an `IsUnit` hypothesis, we would use `isUnit`, not `is_unit`
+- However, since we have an `IsUnit` hypothesis, putting it in the `IsUnit.` 
+  namespace allows for use with dot notation.
+- we should reference the coercion that appears here, which is `Subtype.val`, hence
+  the `val` in the suggested name.
+````
+
+This provides the PR author with a link to the naming conventions in case they
+haven't yet seen it, but also points out the specific issues so that they don't
+have to read through the entire guide again. This is probably helpful also if 
+they have only made one error in the naming convention.
+
+Note: not all declarations have exactly one suitable name, there may be a few, 
+each with their own advantages and disadvantages.
+
+#### Is the PR title and description appropriately informative?
+
+Consider the following PR title and description:
+
+```markdown
+Title: feat(Analysis/SpecificLimits)
+Description: Where should we put these lemmas?
+```
+
+This has two issues: the title doesn't provide any information about the changes
+and the description includes a discussion question, as opposed to information 
+about the changes. A reasonable review comment might be something like:
+
+```markdown
+Please update the PR title and description to be more informative about what 
+you have added or changed as these will be permanently included in the git
+history when this is merged. Questions or topics for discussion are allowed
+in the PR description, but should be placed after the `---`, as then they
+will be treated as comments and not included in the git history.
+```
+
+Of course, you could provide suggestions, or even update the PR title and 
+description yourself, but you may want to point it out, especially if the 
+PR author is a relatively new contributor.
+
+### Documentation
+
+#### do the definitions have sufficiently informative docstrings?
+
+Normally there is a linter to inform the user that *all* definitions should be 
```

Comment:
This is re-enabled now! https://github.com/leanprover-community/mathlib4/commit/34ce2e5a52a64b3d220926c4f4862d6804c4656b